config/default.json

```
@@ -108,5 +108,13 @@
       "encryptionKey": "not-a-real-32-bit-encryption-key",
       "AppInsightsInstrumentationKey":"APPINSIGHTS_INSTRUMENTATIONKEY"
     }
+  },
+  "launchDarkly": {
+    "featureFlag-keys": {
```

missing of camel case and hiphen looks weird
maybe:
```suggestion
    "keys": {
```

not 100% convinced this needs to be in config, they need to be static everywhere a constants file maybe be better